packages/client/src/subscribe/OrderMessages.js

```diff
@@ -55,11 +55,22 @@ export default function OrderMessages(client, options = {}) {
                 if (done) { return }
                 orderingUtil.add(streamMessage)
             }
+        } catch (err) {
+            if (done) { return }
+
+            if (err.code === 'NO_STORAGE_NODES') {
```

Comment:
Do I remember correctly that checking this beforehand doesn't make sense? I.e. it's better to wait for this error to occur than to check for it before commencing OrderingUtil?

Reply:
I mean, either way it's a round trip to the server so might as well *try* i.e. consider this as the check